Godeps/_workspace/src/k8s.io/kubernetes/pkg/admission/plugins.go

```diff
@@ -67,14 +67,15 @@ func RegisterPlugin(name string, plugin Factory) {
 // known. The error is returned only when the named provider was known but failed
 // to initialize. The config parameter specifies the io.Reader handler of the
 // configuration file for the cloud provider, or nil for no configuration.
-func GetPlugin(name string, client clientset.Interface, config io.Reader) (Interface, error) {
+func GetPlugin(name string, client clientset.Interface, config io.Reader) (bool, Interface, error) {
 	pluginsMutex.Lock()
 	defer pluginsMutex.Unlock()
 	f, found := plugins[name]
 	if !found {
-		return nil, nil
+		return false, nil, nil
```

Comment:
> why not just make this return an error like "Unknown admission plugin: %s" and skip the nil check in InitPlugin entirely?

Because errors fatal out in the caller and this isn't an error.